bigtable/client/rpc_backoff_policy.cc

```diff
@@ -55,5 +55,28 @@ std::chrono::milliseconds ExponentialBackoffPolicy::on_completion(
   return delay;
 }
 
+std::unique_ptr<RPCBackoffPolicy> RandomizedBackoffPolicy::clone() const {
+  return std::unique_ptr<RPCBackoffPolicy>(new RandomizedBackoffPolicy(*this));
+}
+
+void RandomizedBackoffPolicy::setup(grpc::ClientContext& /*unused*/) const {}
+
+std::chrono::milliseconds RandomizedBackoffPolicy::on_completion(
+    grpc::Status const& status) {
+  using namespace std::chrono;
+  auto delay = duration_cast<milliseconds>(current_delay_);
+  std::random_device rd;
+  std::mt19937 rng_generator(rd());
+  auto min_value = duration_cast<microseconds>(current_delay_).count();
+  auto max_value = duration_cast<microseconds>(maximum_delay).count();
+  std::uniform_int_distribution<int> rng_distribution(min_value, max_value);
+  auto random_delay = microseconds(rng_distribution(rng_generator));
+  if (random_delay >= maximum_delay_) {
+    random_delay = maximum_delay_;
+  }
+  current_delay_ = random_delay;
```

Comment:
Note that `random_delay` can even be larger than `maximum_delay_` because the distribution never returns a larger value.  That `if()` was unnecessary, though this is now moot because of my request to mix this with exponential backoff.

Reply:
Ack.